github/client.go

```diff
@@ -16,6 +16,15 @@ import (
 
 const (
 	tagsToFetch = 100
+
+	// GitHub API error codes and fields
+	errCodeAlreadyExists = "already_exists"
+	gitTagNameField      = "tag_name"
```

Comment:
I'd separate them as well.
Imagine you have 10 errors and 10 fields.
But already now feels a bit uneasy.